xwiki-platform-core/xwiki-platform-bridge/src/main/java/org/xwiki/bridge/event/DocumentDeletedEvent.java

```diff
@@ -72,6 +78,15 @@ public DocumentDeletedEvent(DocumentReference documentReference)
     public DocumentDeletedEvent(EventFilter eventFilter)
     {
         super(eventFilter);
+        this.sourceReference = null;
+    }
+
+    /**
+     * @return the reference of the deleted document
+     */
+    public DocumentReference getSourceReference()
```

Comment:
This is new public method, so it' missing `@Unstable` and `@since` annotations. Also feels weird, see my previous comment; I'd expect a AbstractDocumentEvent to have a Document source and a getter to get it.